paddle/fluid/eager/api/manual/eager_manual/dygraph_forward_api.h

```diff
@@ -26,6 +26,10 @@ paddle::Tensor conv2d_ad_func(const paddle::Tensor& input,
                               std::vector<int> dilations,
                               int groups,
                               std::string data_format);
+paddle::Tensor multiply_ad_func(const paddle::Tensor& x,
+                                const paddle::Tensor& y);
+paddle::Tensor& multiply__ad_func(paddle::Tensor& x,  // NOLINT
```

Comment:
why inplace?

Reply:
inplace can not be generated because of 'multiply' added in black list